bangazon/bangazon_ui/templates/bangazon_ui/order_detail_view.html

```diff
@@ -1,5 +1,19 @@
-{% extends 'bangazon_ui/index.html'%}
-{% block content %}
-"Hello Are you there?"
+{% extends 'bangazon_ui/index.html'%} {% block content %}
+<h2> ' This is the janky stand in for Order View Template' </h2> {% for item in product_list %}
+<h4> {{item.nam}}</h4>
+<br></br>
+<h4> {{item.price}}</h4> {%endfor %}
+</ul>
```

Comment:
we aren't using an unordered list , i don't think it necessary.  why would we want this ul?